src/Statistics.jl

```diff
@@ -490,6 +487,12 @@ over dimensions. In that case, `mean` must be an array with the same shape as
     Use the [`skipmissing`](@ref) function to omit `missing` entries and compute the
     standard deviation of non-missing values.
 """
+stdm(A::AbstractArray, m; corrected::Bool=true) =
+    sqrt.(varm(A, m; corrected=corrected))
+
+stdm(A::AbstractArray, m::AbstractArray; corrected::Bool=true, dims=:) =
+    sqrt.(varm(A, m; corrected=corrected, dims=dims))
+
 stdm(iterable, m; corrected::Bool=true) =
     std(iterable, corrected=corrected, mean=m)
```

Comment:
Let's use the same approach as above for `std`?
```suggestion
stdm(A::AbstractArray, m::AbstractArray; corrected::Bool=true, dims=:) =
    _std(A, corrected, m, dims)

stdm(iterable, m; corrected::Bool=true) =
    sqrt(var(iterable, corrected=corrected, mean=mean))
```

Reply:
Welp, looks like that broke something. @nalimilan